isaaclab_arena/examples/policy_runner.py

```diff
@@ -35,6 +35,19 @@ def main():
             np.random.seed(args_cli.seed)
             random.seed(args_cli.seed)
 
+        # Post-spawn injection
+        if args_cli.randomize_object_texture_names is not None and len(args_cli.randomize_object_texture_names) > 0:
+            from isaaclab.sim.utils import get_current_stage
+
+            from isaaclab_arena.utils.usd_helpers import randomize_objects_texture
+
+            randomize_objects_texture(
+                object_names=args_cli.randomize_object_texture_names,
+                num_envs=args_cli.num_envs,
+                env_ns=env.scene.env_ns,
+                stage=get_current_stage(),
+            )
```

Comment:
In my opinion, we need to keep variations (and other feature requests) out of the policy_runner.

We should aim for an interface where objects can have texture variations applied. Something like:

```
apple.apply_color_variations()
```

That way, the policy runner doesn't need to worry about this stuff; it's all hidden away in the object definition and the policy runner just runs the policy.